types/three/three-OutlinePass.d.ts

```diff
@@ -0,0 +1,20 @@
+import
+{
+    Camera,
+    Color,
+    Face3,
+    Light,
+    Material,
+    Object3D,
+    Scene,
+    Vector2,
+    Vector3,
+    Vector4
+} from "./three-core";
+export class OutlinePass
+{
+    constructor(size: Vector2, scene: Scene, camer: Camera);
```

Comment:
Mind including the rest of the args here?

Reply:
ok